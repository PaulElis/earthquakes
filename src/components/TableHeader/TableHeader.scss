.TableHeader {
  display: block;
}

.cell{
  color: white;
  background-color: #18dba1;
  border: 1px solid #dddddd;
  text-align: left;
  padding: 8px;
}