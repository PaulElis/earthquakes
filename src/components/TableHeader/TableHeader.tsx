import "./TableHeader.scss";

export default function TableHeader() {
  return (
    <thead>
      <tr data-testid='table-header-row'>
        <td className="cell">ID</td>
        <td className="cell">Time</td>
        <td className="cell">Location</td>
        <td className="cell">Magnitude</td>
        <td className="cell">More</td>
      </tr>
    </thead>
  );
}