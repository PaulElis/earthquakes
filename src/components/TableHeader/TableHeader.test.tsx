import TableHeader from "./TableHeader";

describe("Table Header", () => {
     test('expect TableHeader to match snapshot', () => {
          expect(TableHeader).toMatchSnapshot();
     })
});
