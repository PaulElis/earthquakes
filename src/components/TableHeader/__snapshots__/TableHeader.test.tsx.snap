// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Table Header expect TableHeader to match snapshot 1`] = `[Function]`;
